composer.json

```diff
@@ -92,8 +93,8 @@
             "@cs-check",
             "@test"
         ],
-        "cs-check": "phpcs --colors -p ./src ./tests",
-        "cs-fix": "phpcbf --colors ./src ./tests",
+        "cs-check": "phpcs --colors -p --ignore=*/templates/* ./src ./tests",
```

Comment:
On case ins. file systems this can also effect any path in src with Templates/ dir.
Why do we need it if we already pull templates/ into root?

PS: the `./src ./tests` part should better read `src/ tests/` to be consistent with all the other path declarations we usually do.

Reply:
Not quite sure why I changed this. Will revert.